civihr_employee_portal/tdd/HelperFunctionsTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+require_once __DIR__ . '/../views/civihr_employee_portal.views.inc';
+
+class HelperFunctionsTest extends PHPUnit_Framework_TestCase
+{
```

Comment:
To follow the Drupal code standards, for classes, don't put the opening brace in a newline